docs/_redirects

```diff
@@ -0,0 +1,29 @@
+# Netlify redirects to redirect old tutorial pages to new guide pages
+# These will not work on netlify preview pages
+/tutorial-angular.html            https://videojs.com/guides/angular            301!
+/tutorial-audio-tracks.html       https://videojs.com/guides/audio-tracks       301!
+/tutorial-components.html         https://videojs.com/guides/components         301!
+/tutorial-debugging.html          https://videojs.com/guides/debugging          301!
+/tutorial-embeds.html             https://videojs.com/guides/embeds             301!
+/tutorial-event-target.html       https://videojs.com/guides/event-target       301!
+/tutorial-faq.html                https://videojs.com/guides/faq                301!
+/tutorial-hooks.html              https://videojs.com/guides/hooks              301!
+/tutorial-languages.html          https://videojs.com/guides/languages          301!
+/tutorial-layout.html             https://videojs.com/guides/layout             301!
+/tutorial-live.html               https://videojs.com/guides/live               301!
+/tutorial-middleware.html         https://videojs.com/guides/middleware         301!
+/tutorial-modal-dialog.html       https://videojs.com/guides/modal-dialog       301!
+/tutorial-options.html            https://videojs.com/guides/options            301!
+/tutorial-player-workflows.html   https://videojs.com/guides/player-workflows   301!
+/tutorial-plugins.html            https://videojs.com/guides/plugins            301!
+/tutorial-react.html              https://videojs.com/guides/react              301!
+/tutorial-setup.html              https://videojs.com/guides/setup              301!
+/tutorial-skins.html              https://videojs.com/guides/skins              301!
+/tutorial-tech.html               https://videojs.com/guides/tech               301!
+/tutorial-text-tracks.html        https://videojs.com/guides/text-tracks        301!
+/tutorial-tracks.html             https://videojs.com/guides/tracks             301!
```

Comment:
There's no https://videojs.com/guides/text-tracks. https://videojs.com/guides/text-tracks instead?

Reply:
Probably either remove or port over tutorials-tracks as well (it just links out to video, audio, and text track pages)

Reply:
Oh! That's one that didn't get carried over. We can probably leave it in place because that guide still has content in this repo.